app/scenes/Document/Shared.tsx

```diff
@@ -172,7 +172,7 @@ function SharedDocumentScene(props: Props) {
     }
   }
 
-  if (!response) {
+  if (!response?.sharedTree) {
```

Comment:
In what situation is there a response but not a sharedTree component 🤔

Reply:
It regressed here - on the initial app load, the sharedTree would be undefined until the response is received.
https://github.com/outline/outline/blob/f22bd1d7c8bf25923728f9be41be45bf73315072/app/scenes/Document/Shared.tsx#L132-L134

Reply:
Would it also work to check for `response.document` then? I think that would make more sense if so